packages/drafts-realm/dnd.gts

```diff
@@ -0,0 +1,383 @@
+import {
+  contains,
+  containsMany,
+  linksTo,
+  linksToMany,
+  field,
+  CardDef,
+  Component,
+} from 'https://cardstack.com/base/card-api';
+import StringField from 'https://cardstack.com/base/string';
+import NumberField from 'https://cardstack.com/base/number';
+import BooleanField from 'https://cardstack.com/base/boolean';
+import MarkdownField from 'https://cardstack.com/base/markdown';
+import { fn } from '@ember/helper';
+import { on } from '@ember/modifier';
+
+export class Spell extends CardDef {
+  @field name = contains(StringField);
+  @field level = contains(NumberField);
+  @field isPrepared = contains(BooleanField);
+  @field damageDice = containsMany(StringField); // Field to store dice strings like "2d10", "1d8"
+  @field spellDescription = contains(MarkdownField);
+
+  static embedded = class Embedded extends Component<typeof this> {
+    <template>
+      <strong>{{@model.name}}</strong> (Level: {{@model.level}})
+    </template>
+  };
+
+  static isolated = class Isolated extends Component<typeof this> {
+    // @ts-ignore TS1206: Decorators are not valid here.
```

Comment:
Yea I recall hitting this issue. But im not sure if its tracked in linear

Reply:
I'll add a ticket, there's an annoying combo that the action and tracked imports are also flagged as unused. I assume it's related